index.html

```diff
@@ -111,47 +114,105 @@ <h3 class="project_text">Project name goes here</h3>
             <button type="button" class="collabo">Get my CV <i class="material-symbols-outlined">
-                file_download
-            </i></button>
-               <div class="box_info" id="language">
+                    file_download
+                </i></button>
+            <div class="box_info" id="language">
                 <h3>Languages</h3>
-                 <ul>
+                <ul>
                     <li>JavaScript(ES5/ES6)</li>
                     <li>Ruby</li>
                     <li>HTML</li>
                     <li>CSS</li>
-                 </ul>
-               </div> 
-               <div class="box_info" id="framework">
+                </ul>
+            </div>
+            <div class="box_info" id="framework">
                 <h3>Frameworks</h3>
-                 <ul>
+                <ul>
                     <li>Bootstrap</li>
                     <li>Ruby on Rail</li>
                     <li>RSpec</li>
                     <li>Capibara</li>
                     <li>Selenium</li>
-                 </ul>
-               </div> 
-               <div class="box_info" id="skill">
+                </ul>
+            </div>
+            <div class="box_info" id="skill">
                 <h3>Skills</h3>
-                 <ul>
+                <ul>
                     <li>Codekit</li>
                     <li>Codepen</li>
                     <li>GitHub</li>
                     <li>GitLab</li>
                     <li>Terminal</li>
-                 </ul>
-               </div> 
+                </ul>
+            </div>
         </section>
-        
     </section>
+    <section>
+        <div class="form_img"></div>
+        <div class="form">
+            <h2 class="name">Get Started</h2>
+            <P class="p-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+                ut labore et dolore magna.</P>
+            <form class="contact-form"   action="https://formspree.io/f/mrgdooeg"
+              method="POST">
+                <ul class="footer-list">
+                    <li class="inputfield">
+                        <label>
+                            Your name
+                            <input placeholder="Desmond Wanajala" type="text" name="name" maxlength="30" required />
+                        </label>
+                    </li>
+                    <li class="inputfield">
+                        <label>
+                            Your email
+                            <input placeholder="wanjaladesmond75@gmail.com" type="email" name="email" required />
+                        </label>
+                    </li>
+                    <li>
+                        <textarea name="message" placeholder="Enter your message here..." maxlength="500"
+                            required></textarea>
+                    </li>
+                </ul>
+                <button type="submit" class="collabo">Start collaboration</button>
```

Comment:
- [ ] I think it would be better to make some improvement on this button as listed below :point_down:  and try to match it with that of [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) :+1: :+1: 
    - It would be better to reduce the gap between ```<textarea>``` and the ```button```
**Your design**
![image](https://user-images.githubusercontent.com/102757126/199544572-44388a34-975d-48e0-afe6-c2cd0c93ea47.png)
**Figma design**
![image](https://user-images.githubusercontent.com/102757126/199545155-41b5ff63-a469-45fe-a3a9-cc874366012e.png)
    - It would be better also to improve your button interactions styling to match that of [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) for **active** _(when it's pressed)_, **enable** _(default state)_ and **hover** as shown in the Figma template. :+1: :+1: 
    
       ![image](https://user-images.githubusercontent.com/102757126/199548887-f75bc6a8-b839-459c-9c7e-4e07f6367b45.png)